assets/component-accordion.css

```diff
@@ -32,9 +36,10 @@
 
 .accordion .icon-accordion {
```

Comment:
**Minor bug and a bit of an edge case** 

Now that accordion icons are based on the size of font heading scale, we'd need to limit the width of `.accordion__title` based on that, or it would look like that with long titles:
<img src="https://user-images.githubusercontent.com/37168033/136269748-7eee7566-f553-44e7-9f7b-8d1c29f2f173.png" width="300">

**Expected:**
<img src="https://user-images.githubusercontent.com/37168033/136270318-d0f63a8f-322f-4760-90f5-3c351c2566f6.png)" width="300">


**Solution:**
```css
.accordion__title {
  display: inline-block;
  max-width: calc(100% - 6rem);
```
could be replaced with:
```css
.accordion__title {
  display: inline-block;
  max-width: calc(100% - (var(--font-heading-scale) * 4rem));
```